common/utils/src/main/resources/error/error-classes.json

```diff
@@ -75,6 +75,16 @@
       }
     }
   },
+  "AVRO_INCORRECT_TYPE" : {
+    "message" : [
+      "Cannot convert Avro <avroPath> to SQL <sqlPath> because the original encoded data type is <avroType>, however you're trying to read the field as <sqlType>, which would lead to an incorrect answer. To allow reading this field, enable the SQL configuration: <key>."
```

Comment:
nit: we can hardcode the config name in the error message.

Reply:
sounds good to me!